pages/how-to/set-up/controllers.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Setting Up a Controller
+description: Adding Controllers to Clusters
+---
+
+# Prerequisites
+* **Plural Console `admin` permissions**  
+* **`kubectl` cluster access**
+
+
+# Set Up
+
+### Example: Ingress NGINX
+* **Add a new `HelmRepository` CRD to your _infra_ repo `./apps/repositories`**
+  * Example: `ingress-nginx.yaml`
+```yaml
+apiVersion: source.toolkit.fluxcd.io/v1beta2
```

Comment:
This should use the native helm support not flux-based now that that's fully released.  So this resource isn't necessary, and the `repository` field in the helm block below can just be replaced with`url` pointing to the url in this CRD